### DISCUSSION TITLE
httprunner4可以使用conda环境吗？

### DISCUSSION THREAD

**Discussion by liuxeuman** [httprunner/httprunner#1616] [2 - 答疑（Q&A）]
httprunner4可以使用conda环境吗？目前总是使用.hrp下的虚拟环境，指定venv好像不起作用

**Comment by liuxeuman** [comment]
初学者，已经摸索出怎么使用嘞

**Reply by Xiaojun0822** [reply]
怎么使用呢？可以说下大概步骤么

**Reply by liuxeuman** [reply]
按要求配置好环境，装好模块，使用时切换过去按hrp4命令执行就行了

**Reply by dongli0101** [reply]
指定打开自己的虚拟环境即可，HttpRunnerTest_env是我的虚拟环境【Mac】
hrp startproject demo --venv /Users/dongli/miniforge3/envs/HttpRunnerTest_env

**Reply by liuxeuman** [reply]
或者你不用管 hrp 怎么启动脚本，如果是 python 语言，只把 hrp 当成创建工程的工具，运行脚本用 hrun ./ 运行项目，pytest的配置文件按正常写，需要加的参数 hrun -h 查询
